kmod/src/data.c

```diff
@@ -1912,6 +1912,8 @@ static int scoutfs_data_page_mkwrite(struct vm_area_struct *vma,
 
 	sb_start_pagefault(sb);
 
+	pos = vmf->pgoff << PAGE_SHIFT;
```

Comment:
It looks like this pos = movement has gotten out of sync with the current version of _data_page_mkwrite.  It now adds two pos = assigments, and this moves one, so we're left with two.  I guess we have _page_mkwrite just add one at the top and remove all the pos motion (and commit message) from this.

Reply:
Yeahh this is a rebase accident...